contrib/python-werkzeug/template.py

```diff
@@ -0,0 +1,37 @@
+pkgname = "python-werkzeug"
+pkgver = "3.0.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-flit_core",
+]
+checkdepends = [
+    "python-markupsafe",
+    "python-pytest",
+    "python-pytest-timeout",
+    "python-requests",
+]
+depends = [
+    "python-markupsafe",
+]
+pkgdesc = "WSGI swiss-army knife"
+maintainer = "firefly-cpp <iztok@iztok.space>"
+license = "MIT"
+url = "https://werkzeug.palletsprojects.com"
+source = f"$(PYPI_SITE)/w/werkzeug/werkzeug-{pkgver}.tar.gz"
+sha256 = "507e811ecea72b18a404947aded4b3390e1db8f826b494d76550ef45bb3b1dcc"
+
+
+def post_extract(self):
+    # tests require ephemeral-port-preserve
+    self.rm("tests/conftest.py")
+    self.rm("tests/test_debug.py")
+    # tests require watchdog
+    self.rm("tests/test_serving.py")
+    self.rm("tests/middleware/test_http_proxy.py")
```

Comment:
`make_check_args = ["--deselect", "tests/....` instead of removing files

Reply:
@nekopsykose, skipping tests is not an option since scripts import the missing packages. Interestingly, --ignore flag in make_check_args is not working

Reply:
see current version of template

Reply:
> skipping tests is not an option

> Interestingly, --ignore flag in make_check_args is not working

works fine though (though yes, need ignore instead of deselect). conftest.py is a special pytest import that defines fixtures, so you need

```
make_check_args = [
    "--noconftest",
    # need ephemeral-port-preserve + watchdog
    "--ignore=tests/middleware/test_http_proxy.py",
    "--ignore=tests/test_debug.py",
    "--ignore=tests/test_serving.py",
]
```

Reply:
wonderful; thanks